client/src/views/workspaces/WorkspaceSharingModal.vue

```diff
@@ -584,38 +639,86 @@ async function onBatchRoleChange(newRoleOption: MsOption): Promise<void> {
     max-height: 2.25rem;
     max-width: 15rem;
     margin: 0;
+
+    @include ms.responsive-breakpoint('sm') {
+      max-width: 100%;
+    }
   }
 
   &-right {
     display: flex;
+    align-items: center;
     gap: 1rem;
 
+    @include ms.responsive-breakpoint('sm') {
+      position: absolute;
+      left: 0;
+      bottom: 0;
+      width: 100%;
+      padding: 1rem 1.5rem 2rem;
+      justify-content: space-between;
+      background: var(--parsec-color-light-secondary-white);
+      border-radius: var(--parsec-radius-8) var(--parsec-radius-8) 0 0;
+      box-shadow: var(--parsec-shadow-strong);
+      z-index: 2;
+    }
+
+    .dropdown {
+      border: 1px solid var(--parsec-color-light-secondary-medium);
+      border-radius: var(--parsec-radius-8);
+    }
+
     #batch-activate-button {
       font-size: 0.875rem;
-      padding: 0.5rem 1rem;
+      padding: 0.625rem 1rem;
       color: var(--parsec-color-light-secondary-text);
-      background: var(--parsec-color-light-secondary-premiere);
-      border-radius: var(--parsec-radius-6);
+      border: 1px solid var(--parsec-color-light-secondary-medium);
+      border-radius: var(--parsec-radius-8);
       cursor: pointer;
       transition: background 0.2s;
+      display: flex;
+      align-items: center;
+      margin-left: auto;
+      gap: 0.375rem;
+
+      @include ms.responsive-breakpoint('sm') {
+        order: 1;
+      }
+
+      .checkbox-icon {
+        color: var(--parsec-color-light-secondary-text);
+        font-size: 1rem;
```

Comment:
Icon is a bit small and pixelized
![Capture d’écran 2025-04-30 à 15 50 37](https://github.com/user-attachments/assets/cfd08ed5-bc63-4763-80e6-fa0701f90b48)